### PR TITLE
Changed the actual path and filename format style

### PR DESCRIPTION
**Pull request by ghost**
https://bugzilla.redhat.com/show_bug.cgi?id=1491134

### CODE REVIEW

**Comment by tmorriso-rh** [comment]
@openshift/team-documentation PTAL.

**Comment by ahardin-rh** [comment]
@tmorriso-rh The changes LGTM. If this is for 3.6 only, then you may need to submit this as a new PR (as feature branch) against the 3.6 branch. According to the comments in the BZ, however, Weihua Meng noted that it worked for 3.5 -> 3.6, 3.6 -> 3.7, 3.7 -> 3.9 so I assume this would also apply to  3.7 and 3.9, which means you would be okay to just add the appropriate labels here, but I would wait for confirmation and ensure that this is okay to also go into the master branch (Origin docs). If not, we will need to add ifdef statements.

File: install_config/downgrade.adoc [hunk 1]
```diff
@@ -38,7 +38,7 @@ etcd data directory. Ensure these exist on your masters and etcd members:
 
 ----
 /etc/origin/master/master-config.yaml.<timestamp>
-/var/lib/etcd/openshift-backup-<timestamp>
+/var/lib/etcd/openshift-backup-pre-upgrade-20180207123542
```
Comment (wmengRH) [inline install_config/downgrade.adoc]: /var/lib/etcd/openshift-backup-pre-upgrade-<timestamp>
looks more accurate and style consistent

**Comment by tmorriso-rh** [comment]
@vikram-redhat This pull request has been reviewed (QE/SME/peer) and merged. This needs to be cherry picked to 3.6, 3.7, and 3.9.

**Comment by tmorriso-rh** [comment]
No rev history needed.

**Comment by tmorriso-rh** [comment]
Thanks @ahardin-rh. I checked the topic map, and it states that this topic is only for the openshift-enterprise docs.

**Comment by tmorriso-rh** [comment]
Creating a new PR for the 3.7 and 3.9 branches.

**Pull request merged**